### DISCUSSION TITLE
Vercel "Out of Memory" ("OOM") error

### DISCUSSION THREAD

**Discussion by stephenasuncionDEV** [vercel/community#4922] [Bug Report]
### Page to Investigate

https://vercel.com/interax-ai-team/interxai-web/uXD7gNntCjKfWCSeX2h2q5BNEiCf

### Steps to Reproduce

I was trying to build a preview env for one of my website but I'm getting this out of memory error.

![284468932-70a17c6f-483c-45e7-862d-756f37d7e413](https://github.com/vercel/community/assets/73958774/3fdb771c-3faa-4981-a5fd-84106539736f)

Switched to pnpm, got only 4 errors now:
![image](https://github.com/vercel/community/assets/73958774/25727e02-9a73-48f4-be1d-02aa9ea0f71d)

**Comment by stephenasuncionDEV** [comment]
Can confirm that the error is not in the config because I could deploy it with no problem if I empty out the src folder with just 1 page.

**Comment by amyegan** [comment]
Hey @stephenasuncionDEV. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) we can use to help with debugging?
